_posts/2022-02-02-#748.md

```diff
@@ -10,19 +10,50 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- [libsecp256k1](https://github.com/bitcoin-core/secp256k1/) is the library that Bitcoin Core uses for low-level cryptographic operations such as signing and verification.
+
+- This PR adds usage examples for several common use cases: ECDSA signatures, Schnorr signatures, and (elliptic curve) Diffie-Hellman key exchange. There are no existing usage examples in the library.
+
+- The examples are intended to clarify the API by supplementing the documentation in the [various header files](https://github.com/bitcoin-core/secp256k1/tree/master/include).
+
 ## Questions
+
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
--->
+2. Can you compile and run the example? Note that the examples are not compiled by default. Therefore, you must run `./configure` with the `--enable-examples` arguments. Moreover, the ECDH and Schnorr signature examples are only compiled if the modules are enabled, which is done by providing the `--enable-module-ecdh` and `--enable-experimental --enable-module-schnorrsig` arguments to `./configure`.
+3. Why do the examples demonstrate how to obtain randomness? Is this a good idea?
+4. Are those the correct ways to obtain randomness on each of the supported operating systems?
+5. Can you follow the examples? Is it clear how they should be generalized to a production system?
+6. Is there anything missing in the examples (e.g. `context_randomize`, `ec_seckey_verify`, return value check, setting secrets to 0, etc...)?
+7. Are the comments in the examples helpful?
+8. Are the comments sufficient (e.g. ECDSA takes message hash, not message)?
+9. Is it a good idea to retry (in a loop) operations that could only fail with negligible probability?
```

Comment:
Perhaps link to the comment or code https://github.com/bitcoin-core/secp256k1/pull/748#pullrequestreview-404120824 so attendees have some context

Reply:
Good idea. Done.